multimodal/tests/unittests/test_predictor.py

```diff
@@ -525,6 +529,8 @@ def test_model_configs():
                 "ffn_activation": "geglu",
                 "head_activation": "relu",
                 "data_types": None,
+                "additive_attention": False,
+                "share_qv_weights": False,
```

Comment:
No test for `additive_attention ` and `share_qv_weights ` being True.

Reply:
Should I just set "additive_attention" and "share_qv_weights" to be True for fusion transformer? or create new test cases for these two parameters?

Reply:
New test cases would be better. We could remove `clip` in `test_model_config` to accelerate the tests.

Reply:
I think the test case for the new configuration is within ```test_model_config```, since it was added as an argument to ```FT_Transformer```.